subsys/dm/Kconfig

```
@@ -47,68 +47,68 @@ config DM_HIGH_PRECISION_CALC
 	default n
 	bool "Use high-precision distance estimation"
 	help
-		"Use a more compute-intensive algorithm for the distance estimation.
-		Only works with MCPD ranging mode"
+	  Use a more compute-intensive algorithm for the distance estimation.
+	  Only works with MCPD ranging mode
 
 config DM_TIMESLOT_RESCHEDULE
 	bool "Timeslot reschedule"
 	default n
 	help
-	  "Enable allocation of the next timeslot after the previous timeslot has succeeded."
+	  Enable allocation of the next timeslot after the previous timeslot has succeeded.
 
 config DM_INITIATOR_DELAY_US
 	int "Initiator start delay"
 	default 1000
 	help
-	  "Additional Initiator Start delay time. Used to adjust synchronization."
+	  Additional Initiator Start delay time. Used to adjust synchronization.
```

```suggestion
	  Additional Initiator Start delay time.
	  Used to adjust synchronization.
```
Not sure whether these need to be on separate lines, but that seems to be the case with other instances.

---

Good point.